src/ring_theory/jacobson.lean

```diff
@@ -18,6 +19,10 @@ The following conditions are equivalent for a ring `R`:
 
 Any ring satisfying any of these equivalent conditions is said to be Jacobson.
 
+Some particular examples of Jacobson rings are also proven.
+`is_jacobson_quotient` says that the quotient of a jacobson ring is jacobson.
+`is_jacobson_localization` says the localization of a jacobson ring to a single element is jacobson.
```

Comment:
Let's make the capitalization consistent (AFAIK, standard usage is with a capital letter).
```suggestion
`is_jacobson_quotient` says that the quotient of a Jacobson ring is Jacobson.
`is_jacobson_localization` says the localization of a Jacobson ring to a single element is Jacobson.
```

Reply:
The switch to lowercase was made on the previous PR in order to match with the capitalization of the mathlib definitions. I don't really have an opinion either way so I'm fine switching back to capitalized to match with general usage.